### DISCUSSION TITLE
How to specify the destination location for the msi file binaries in a WiXSetup Project?

### DISCUSSION THREAD

**Discussion by sansa20** [wixtoolset/issues#7390] [Questions] [answered]
I have a WiXSetup project that I am using to generate an msi file with a few binaries. the msi file is created upon building the project. But when I run the msi files, I want the binaries to be created in a specific location in file explorer, How do I make this happen? I believe it has something to do with the directory element? This is my code right now, for reference- it is essentially the default.
```

<Fragment>
		<Directory Id="TARGETDIR" Name="SourceDir">
			<Directory Id="ProgramFilesFolder">
				<Directory Id="INSTALLFOLDER" Name="OpenTapWixSetup" />
			</Directory>
		</Directory>
	</Fragment>
```
Is it possible for me to change it in such a manner that when I run the msi file, the binaries are generated in a specific location in file explorer

**Comment by robmen** [comment, marked as answer]
This is a pretty basic question. I suggest reading through the [WiX Tutorial from FireGiant](https://www.firegiant.com/wix/tutorial/) or watching the [Deployment Dojo episodes](https://bit.ly/github-discussions-dojo-playlist) to get a foundational understanding of how the WiX Toolset works. This [episode of the Deployment Dojo](https://www.youtube.com/live/y14Xd6VJJAw?feature=share) would be most applicable.

**Comment by chrpai** [comment]
I have custom tooling which is FOSS that helps you get started with WiX.  I also have tutorials and can provide 1hr complimentary mentoring and consulting services to do it for you.  FireGiant offers a lot of that also.
